ci/ccp_ci_unittests.sh

```diff
@@ -49,29 +49,36 @@ echo "=========================================================="
 sshopts="-t -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -l root"
 ssh_cmd="ssh $sshopts $CICO_hostname"
 
+echo "====Installing necessary packages for running tests on the duffy node===="
 # install the needed packages on the duffy node which will the tests
-$ssh_cmd "yum -y install epel-release && yum -y install rsync git PyYAML python-networkx python2-nose"
+$ssh_cmd "yum -y install epel-release && yum -y install rsync git PyYAML python-networkx python2-nose" >> /tmp/unittests_setup_log.txt
 package_installed_success=$?
 
-if [ $package_installed_success -ne 0]
+if [ $package_installed_success -ne 0 ]
 then
-
+  echo "==================Package Installation Logs============"
+  cat /tmp/unittests_setup_log.txt
+  echo "======================================================="
   mark_failure "Failed to install required packages on duffy node, exiting!"
 fi
+echo "====Duffy node is setup with necessary dependencies===="
 
-
+echo "======Copying source code to the duffy node======"
 # sync the codebase from PR to duffy node
 rsync -e "ssh $sshopts" -Ha $(pwd)/ $CICO_hostname:payload
 rsync_success=$?
 
-if [ $rsync_success -ne 0]
+if [ $rsync_success -ne 0 ]
 then
   mark_failure "Failed to rsync the PR source code on duffy node, exiting!"
```

Comment:
Shouldn't we store output of `rsync` to a file similar to what we are doing while installing packages in above step and then do a `cat` of the file?

Reply:
yeah, good idea, we can show the logs in case of failure